debezium-server/debezium-server-ybexporter/src/main/java/io/debezium/server/ybexporter/ExportStatus.java

```diff
@@ -272,7 +272,7 @@ public void queueSegmentCreated(long segmentNo, String segmentPath){
         Statement insertStmt;
         try {
             insertStmt = metadataDBConn.createStatement();
-            insertStmt.executeUpdate(String.format("INSERT OR IGNORE into %s VALUES(%d, '%s', 0)", QUEUE_SEGMENT_META_TABLE_NAME, segmentNo, segmentPath));
+            insertStmt.executeUpdate(String.format("INSERT OR IGNORE into %s VALUES(%d, '%s', 0, 0, 0, 0)", QUEUE_SEGMENT_META_TABLE_NAME, segmentNo, segmentPath));
```

Comment:
Actually, can you change the statement to include the column names? That would make it explicit.

Reply:
You could just ignore the extra columns since they seem to have defaults.

Reply:
Done